### DISCUSSION TITLE
How to change Font wieight and Font size

### DISCUSSION THREAD

**Discussion by tienpham212** [add2cal/add-to-calendar-button#376] [Q&A]
Hi I tried to change font weight and font size. I have an issues with it try different way like use customCss by https://github.com/tienpham212/Weeding/blob/main/wedding-invitation/src/Component/Time/Time.css and also try to do "../Time.css" but both not work. The only working for me is try to do like this 
styleLight="--btn-background: #b2d4ef; --font: 'Montserrat Alternates', sans-serif; --base-font-size-l: 50px;"
but it do not have the option to change font weight and font size. 
Do we have any way to do that? or can you add that feature?
Many Thanks

**Comment by jekuer** [comment]
Hi @tienpham212 ,

Thanks for your input.
Offering font-weight as variable for the styleLight options sounds fair and will become part of a future release.
However, I do not get, what you mean with font-size. You can use `--base-font-size-l`, `--base-font-size-m`, and `--base-font-size-s` for this. The l, m, s refers to screen sizes. The reason it is called "base" is that it can be enlarged or downsized by the `size` option.
